articles/aks/operator-best-practices-cluster-security.md

```diff
@@ -53,6 +53,10 @@ For more information about Azure AD integration, Kubernetes RBAC, and Azure RBAC
 >
 > Add a network policy in all user namespaces to block pod egress to the metadata endpoint.
 
+> [!NOTE]
+> To implement Network Policy, include the attribute --network-policy azure when creating the AKS cluster. Use the following command to create the cluster
```

Comment:
As per my original request for change, need to enclose the attribute --network-policy azure in tick marks (i.e. code block format). Also need to add a colon after the word "cluster" before the example command.

Reply:
completed